tests/Services/MediaCounterServiceS3Test.php

```
@@ -0,0 +1,168 @@
+            sleep(1);//to void overriding the file names due to time()
+            Storage::disk('entry_original')->put($this->project->ref . '/' . $this->entry->uuid . '_' . time() . '.jpg', '');
+            Storage::disk('entry_thumb')->put($this->project->ref . '/' . $this->entry->uuid . '_' . time() . '.jpg', '');
+        }
+
+        $mediaCounterService = new MediaCounterService();
+        $counters = $mediaCounterService->countersMedia($this->project->ref);
+        $this->assertEquals($numOfPhotos * 2, $counters['counters']['total']);
+        $this->assertEquals($numOfPhotos * 2, $counters['counters']['photo']);
+        $this->assertEquals(0, $counters['counters']['audio']);
+        $this->assertEquals(0, $counters['counters']['video']);
+    }
```

PHPMD (2.15.0)

The method test_count_media_s3_with_photos is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)